tools/apprunner-gradle-plugin/src/main/java/org/projectnessie/quarkus/gradle/QuarkusAppPlugin.java

```diff
@@ -18,27 +18,58 @@
 import org.gradle.api.Action;
 import org.gradle.api.Plugin;
 import org.gradle.api.Project;
+import org.gradle.api.Task;
 import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.tasks.testing.Test;
 
+@SuppressWarnings("Convert2Lambda") // Gradle complains when using lambdas (build-cache won't wonk)
 public class QuarkusAppPlugin implements Plugin<Project> {
 
+  static final String START_TASK_NAME = "nessie-quarkus-start";
+  static final String STOP_TASK_NAME = "nessie-quarkus-stop";
+  static final String EXTENSION_NAME = "nessieQuarkusApp";
+  static final String CONFIG_NAME = "nessieQuarkusRunner";
```

Comment:
similar as comment above

Reply:
I've renamed those to make clear that this comes from the nessie-quarkus-gradle plugin and not from the "original" Quarkus plugin, which seems to have changed quite a bit recently. The Iceberg build script has no more references to any of these.